### ISSUE TITLE
Configurable 'use strict' on generator JS files

### ISSUE THREAD

**Issue by joeljeske** [cgross/generator-cg-angular#92]
It would be nice to allow generating files with the 'use strict' directive within functions. It would be useful to allow a choice during initial setup to generate files with or without the 'use strict' directive.

**Comment by pusherman** [comment]
You can always just override the templates that are generated to include 'use strict'

**Comment by joeljeske** [comment]
Yes that's very true. I thought it might be handy to allow an option for this as opposed to having to edit the config file and find the existing templates and adding strict to them.  It also prompts the user to think if they want to use strict (which is better practice). It also helps avoiding making the global scope strict which is another common bad practice (especially when the build concatenates all the files together automatically).

**Comment by SirRagnar** [comment]
+1
